apps/passport-server/migrations/48_topic_pkey.sql

```diff
@@ -0,0 +1,23 @@
+-- Add a unique index instead of a primary key constraint
+ALTER TABLE telegram_chat_topics DROP CONSTRAINT telegram_chat_anon_topics_pkey;
+
+-- Make topic id nullable
+ALTER TABLE telegram_chat_topics ALTER COLUMN topic_id DROP NOT NULL;
```

Comment:
Probably move this to the top and combine the line above with the line below into a single block.

Reply:
Added more comments